src/pages/home/report/ReportActionItem.js

```diff
@@ -452,26 +452,32 @@ function ReportActionItem(props) {
         const parentReport = ReportActionsUtils.getParentReportAction(props.report);
         if (ReportActionsUtils.isTransactionThread(parentReport)) {
             return (
-                <MoneyRequestView
-                    report={props.report}
-                    shouldShowHorizontalRule={!props.shouldHideThreadDividerLine}
-                />
+                <OfflineWithFeedback pendingAction={props.action.pendingAction}>
```

Comment:
@grgia We use this condition for setting `pendingAction`. Does it make sense to add here too.

https://github.com/Expensify/App/blob/c9df4dbfd4c8f6dc97b574ae9139714c10380ce8/src/pages/home/report/ReportActionItem.js#L535

Reply:
I don't think we need to do that because they can't be edited. This is the PR where we added that https://github.com/Expensify/App/pull/12426